File: app/src/main/java/com/mgaetan89/showsrage/model/Serie.java
package com.mgaetan89.showsrage.model;

import com.google.gson.annotations.SerializedName;

public class Serie {
	@SerializedName("Actors")
	private String actors;

	@SerializedName("Awards")
	private String awards;

	@SerializedName("Country")
	private String country;

	@SerializedName("Director")
	private String director;

	@SerializedName("Genre")
	private String genre;

	@SerializedName("imdbID")
	private String imdbId;

	private String imdbRating;

	private String imdbVotes;

	@SerializedName("Language")
	private String language;

	@SerializedName("Metascore")
	private String metascore;

	@SerializedName("Plot")
	private String plot;

	@SerializedName("Poster")
	private String poster;

	@SerializedName("Rated")
	private String rated;

	@SerializedName("Released")
	private String released;

	@SerializedName("Response")
	private String response;

	@SerializedName("Runtime")
	private String runtime;

	@SerializedName("Title")
	private String title;

	@SerializedName("Type")
	private String type;

	@SerializedName("Writer")
	private String writer;

	@SerializedName("Year")
	private String year;

	public String getActors() {
		return this.actors;
	}

	public String getAwards() {
		return this.awards;
	}

	public String getCountry() {
		return this.country;
	}

	public String getDirector() {
		return this.director;
	}

	public String getGenre() {
		return this.genre;
	}

	public String getImdbId() {
		return this.imdbId;
	}

	public String getImdbRating() {
		return this.imdbRating;
	}

	public String getImdbVotes() {
		return this.imdbVotes;
	}

	public String getLanguage() {
		return this.language;
	}

	public String getMetascore() {
		return this.metascore;
	}

	public String getPlot() {
		return this.plot;
	}

	public String getPoster() {
		return this.poster;
	}

	public String getRated() {
		return this.rated;
	}

	public String getReleased() {
		return this.released;
	}

	public String getResponse() {
		return this.response;
	}

	public String getRuntime() {
		return this.runtime;
	}

	public String getTitle() {
		return this.title;
	}

	public String getType() {
		return this.type;
	}

	public String getWriter() {
		return this.writer;
	}

	public String getYear() {
		return this.year;
	}
}

File: app/src/main/kotlin/com/mgaetan89/showsrage/model/Serie.kt
package com.mgaetan89.showsrage.model

import com.google.gson.annotations.SerializedName
import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class Serie : RealmObject() {
	@SerializedName("Actors")
	open var actors: String? = null
	@SerializedName("Awards")
	open var awards: String? = null
	@SerializedName("Country")
	open var country: String? = null
	@SerializedName("Director")
	open var director: String? = null
	@SerializedName("Genre")
	open var genre: String? = null
	@PrimaryKey
	@SerializedName("imdbID")
	open var imdbId: String? = null
	open var imdbRating: String? = null
	open var imdbVotes: String? = null
	@SerializedName("Language")
	open var language: String? = null
	@SerializedName("Metascore")
	open var metascore: String? = null
	@SerializedName("Plot")
	open var plot: String? = null
	@SerializedName("Poster")
	open var poster: String? = null
	@SerializedName("Rated")
	open var rated: String? = null
	@SerializedName("Released")
	open var released: String? = null
	@SerializedName("Response")
	open var response: String? = null
	@SerializedName("Runtime")
	open var runtime: String? = null
	@SerializedName("Title")
	open var title: String? = null
	@SerializedName("Type")
	open var type: String? = null
	@SerializedName("Writer")
	open var writer: String? = null
	@SerializedName("Year")
	open var year: String? = null
}
